stores/useWalletStore.tsx

```diff
@@ -108,9 +108,10 @@ const INITIAL_REALM_STATE = {
   councilTokenOwnerRecords: {},
   loading: true,
   mints: {},
+  // @askfish: this should probably just be undefined, and leave it up to components how to handle things while loading
   programVersion: 1,
```

Comment:
echoing my comment here. When stuff is loading we shouldn't just guess the value. If components or functions want to do that then they should do that. But fixing this was too far reaching for this PR.